tools/generator/test/CreateXCSchemesTests.swift

```diff
@@ -0,0 +1,184 @@
+import PathKit
+import XcodeProj
+import XCTest
+
+@testable import generator
+
+class CreateXCSchemesTests: XCTestCase {
+    let filePathResolver = FilePathResolver(
+        internalDirectoryName: "rules_xcodeproj",
+        workspaceOutputPath: "examples/foo/Foo.xcodeproj"
+    )
+    let pbxTargetsDict: [TargetID: PBXTarget] =
+        Fixtures.pbxTargetsWithDependencies(
```

Comment:
I don't think you need dependencies set here, just `Fixtures.pbxTargets` should suffice.

Reply:
Done